example_cpp_smart_card_client_app/build/Makefile

```diff
@@ -30,6 +30,11 @@ include $(THIRD_PARTY_DIR_PATH)/pcsc-lite/webport/js_client/include.mk
 include $(THIRD_PARTY_DIR_PATH)/pcsc-lite/webport/cpp_client/include.mk
 
 
+ifeq (app,$(PACKAGING))
```

Comment:
This has to be reverted I think - or otherwise the build scripts need to be improved to skip building examples in the "app" mode...